app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Here is the list of all users</h1>
+<div class="container">
+<h1 class="heading">RailsBlog</h1>
+<% @users.each do |user|%>
+<a href="/users/<%= user.id%>">
+  <div class="users-list">
+    <img src="<%= user.photo %>" alt="User" />
+    <div class="description">
+      <h2> <%= user.name %></h2>
+      <h4>Number of posts: <%= user.posts.count %></h4>
+    </div>
+  </div>
+  </a>
+<%end%>
```

Comment:
[REQUIRED] The best practice is to use partials in scenarios like this Kindly read up on rails partials.

Reply:
Thank you, I will implement this right away.